src/components/ReadyForBillBtn.js

```diff
@@ -22,8 +22,7 @@ class ReadyForBillBtn extends Component {
         null,
         dom.button(
           {
-            className: 'order-entries ready-for-bill btn btn-primary btn-lg btn-block',
-            disabled: !this.props.overrideDisable && this.shouldBeDisabled(),
+            className: `order-entries ready-for-bill btn btn-primary btn-lg btn-block ${ this.shouldBeHidden() ? 'hidden' : 'show' }`,
```

Comment:
These class names are getting pretty unwieldy huh? We should consider looking into this: https://github.com/JedWatson/classnames